js/test/components/home.test.tsx

```diff
@@ -1,6 +1,11 @@
 import { Home } from "../../components/home";
 import { render } from "@testing-library/react";
 
+jest.mock("../../hooks/useEmployees", () => ({
+  __esModule: true,
```

Comment:
I was noticing in mocking fun this morning that this line may not be required if there's no default/non-named export involved, so maybe try it without?

Reply:
Oh, neat, let me take a look.

Reply:
0118b17